### DISCUSSION TITLE
Trigger build

### DISCUSSION THREAD

**Discussion by emarj** [Cockpit-HQ/Cockpit#83] [General]
Hi there,

I have just found out about Cockpit, it seems great but I did not try it yet. After having a look at the website I have one question.

With the base/free version is there the possibility to trigger a build in a CI environment (GitHub, Netlify, etc.) after editing the content? In other words, can this be used in conjunction with Static Site Generators?

Thanks a lot.

**Comment by aheinze** [comment]
Hi!

Sorry for the late reply. Yes, this is possible. Create the following file `/config/bootstrap.php` with the following contents:

```php 
<?php

$app->on('content.item.save', function() {

 // trigger your CI, eg via curl 

});

```
